### PR TITLE
Add many new things

### PR DESCRIPTION
**Pull request by williambl**

### CODE REVIEW

File: data/discord [hunk 1]
```diff
@@ -21,6 +24,8 @@ the Minecraft Cursed Legacy discord
 the FabLabs discord
 the Furbic discord
```
Comment (sylv256) [inline data/discord:25]: there’s a furbric discord now? :help_me:

File: data/person [hunk 2]
```diff
@@ -83,3 +83,9 @@ Acrogenous
 Whangd00dle
 Will BL
 Tehc
```
Comment (sylv256) [inline data/person:85]: Add “Voxel” too (they’re a bot)

Comment (williambl) [reply data/person:85]: Which server?

File: data/discord (hunk 1 above)
Comment (williambl) [reply data/discord:25]: Idk, didn't add that one :p

File: data/reaction [hunk 3]
```diff
@@ -7,6 +7,5 @@
 . %relevantperson gets unbanned from %community.
 . %relevantperson loses a lot of %internetpoints
 . %relevantperson rehosts $mod:mentioned on %hostingsite.
-. %reactor creates a fork of $mod:mentioned and swaps it to MCP mappings.
```
Comment (frqnny) [inline data/reaction:10]: Why did you delete this one?

Comment (williambl) [reply data/reaction:10]: `alternatemappings` contains MCP
